components/02-elements/form/select/select.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     label: {
       attribute: '',
       text: 'Label text',
+      class: 'select__label'
     },
     options: [
       {
```

Comment:
In `select.hbs` there some work to do: 
 - move `for` value  to config and add `id` (value set in config)
 - move `name` value to config
 - Generally b/c we use some js select is not accessible. But it could take some time to customize is, I've added to an issue https://github.com/SnowdogApps/magento2-alpaca-components/issues/133

`File upload` element is not fully accessible neither, I'll put a comment in the issue

Reply:
Sure. I was just updating styles here mainly. Changed first two thinkgs from the bullet list above.